wagtaildraftail/client/wagtaildraftail.js

```diff
@@ -54,4 +64,18 @@ const initDraftailEditor = (fieldName, options = {}) => {
 
 window.initDraftailEditor = initDraftailEditor;
 
+window.wagtaildraftail = {
+  // Expose registry methods
+  registerDecorator: registerDecorator,
+  getDecorator: getDecorator,
+  registerSource: registerSource,
+  getSource: getSource,
+  registerStrategy: registerStrategy,
+  getStrategy: getStrategy,
+
+  // Expose basic React methods for basic needs
+  createClass: React.createClass,
+  createElement: React.createElement,
```

Comment:
See my reply to the similar comment below on `tests/testapp/home/static/button_entity.js`.